.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,4 +7,8 @@
 3. [STEP_3]
 
 ### Review
-@[USER_NAME]
\ No newline at end of file
+@[USER_NAME]
+
+Make sure strings can be translated:
```

Comment:
Nit but, can we change the `can` to `will`? With the `can` it's not clear whether the author needs to do something special with the string formats perhaps, or something.

While at it, I understand that the copy here is probably intentionally generalized but I wonder, how about we mention the WordPress app specifically?

Suggestion: "Make sure strings will be translated by the WordPress Android app"

Reply:
I updated it, but I omitted the last part because this is a library and could be used by other clients... wdyt?

Reply:
Thanks for the updates!

I think that the whole prompt to update the client's `strings.xml` is very specific to the WordPress app anyway (other clients might have a different infrastructure for translations) so generalizing it might be confusing for the PR author. That said, I think this is not a blocker for the PR so, let's run with the current version 👍.